app/models/grda_warehouse/warehouse_reports/hud_lot.rb

```diff
@@ -233,7 +233,7 @@ def details_by_range
       self_report_dates
     end
 
-    private def set_unknowns(un_processed_dates) # rubocop:disable Do not prefix writer method names with `set_`. (convention:Naming/AccessorMethodName
+    private def set_unknowns(un_processed_dates) # not prefix writer method names with `set_`. (convention:Naming/AccessorMethodName
```

Comment:
seems like we were trying for  a magic comment to disable the linter here
```suggestion
    private def set_unknowns(un_processed_dates) # rubocop:disable Naming/AccessorMethodName
```